plugins/ruby-foreman-ansible/debian/control

```diff
@@ -2,18 +2,17 @@ Source: ruby-foreman-ansible
 Section: ruby
 Priority: extra
 Maintainer: Daniel Lobato Garcia <elobatocs@gmail.com>
-Build-Depends: debhelper, foreman (>= 1.24.0~rc1), ruby-foreman-deface (<<2.0.0),
-  foreman-assets(>= 1.24.0~rc1), ruby-foreman-tasks (>= 0.8.2-2), ruby-foreman-tasks (<<1.0.0),
-  ruby-foreman-remote-execution (>= 2.0.0-1), ruby-foreman-remote-execution (<<3.0.0),
-  foreman-sqlite3 (>= 1.24.0~rc1)
+Build-Depends: debhelper, foreman (>= 2.0.0~rc1), ruby-foreman-deface (<<2.0.0),
```

Comment:
Looks like this is failing because we don't have 2.0.0~rc1 yet

Reply:
We have that in nightly. :)